src/Evolution/Systems/Cce/ExtractionRadius.hpp

```diff
@@ -0,0 +1,19 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <optional>
+#include <string>
+
+namespace Cce {
+// retrieves the extraction radius from the specified file.
```

Comment:
Add another slash to make these Doxygen comments?

And maybe group together with next one?